shared/tracker/proofs.render.desktop.js

```diff
@@ -65,7 +65,7 @@ class ProofsRender extends Component {
   }
 
   _isTracked (proof: Proof): boolean {
-    return this.props.currentlyFollowing && (!proof.meta || proof.meta === metaNone)
+    return this.props.currentlyFollowing && (!proof.meta || proof.meta === metaNone || proof.meta === metaWhatevz)
```

Comment:
seems like this kind of stuff should go up a level into the reducer or something, let's revisit when we make these on mobile and start hoisting more of this logic up